app/src/main/java/com/asfoundation/wallet/ui/balance/RestoreWalletPresenter.kt

```diff
@@ -49,6 +52,14 @@ class RestoreWalletPresenter(private val view: RestoreWalletView,
   private fun handleRestoreFromFile() {
     disposable.add(view.restoreFromFileClick()
         .doOnNext { activityView.askForReadPermissions() }
+        .doOnNext {
+          walletEventSender.sendWalletImportRestoreEvent(WalletAnalytics.ACTION_IMPORT_FROM_FILE,
+              WalletAnalytics.STATUS_SUCCESS)
+        }
+        .doOnError { t ->
+          walletEventSender.sendWalletImportRestoreEvent(WalletAnalytics.ACTION_IMPORT_FROM_FILE,
+              WalletAnalytics.STATUS_FAIL, t.message)
+        }
```

Comment:
I don't know if this is suppose to go here, we are just asking for permission here